enclave_wrangler/nameBasedEndpoints.json

```diff
@@ -0,0 +1,5276 @@
+{
```

Comment:
New API endpoint reference: I saved the response w/ the new endpoints as JSON in enclave wrangler for reference. This isn't really part of the main purpose of the PR. If we don't want this here though, can remove as well.